release/NativeEditPlugin/Plugins/iOS/EditBox_iOS.m

```diff
@@ -190,6 +194,21 @@ -(void) setRect:(JsonObject*)json
     editView.frame = CGRectMake(x, y, width, height);
 }
 
+-(void) setTextSize:(JsonObject*)json
+{
+    float fontSize = [json getFloat:@"fontSize"];
+    // Conversion for retina displays
+    fontSize = fontSize / [UIScreen mainScreen].scale;
+    
+    if([editView isKindOfClass:[UITextField class]]) {
+        UITextField *textField = ((UITextField*)editView);
+        UIFont *newFont = [[textField font] fontWithSize:fontSize];
+    } else if([editView isKindOfClass:[UITextView class]]){
+        UITextView *textView = ((UITextView*)editView);
+        UIFont *newFont = [[textView font] fontWithSize:fontSize];
+    }
+}
```

Comment:
Aren't we missing a setFont call?

Reply:
Please fix the code style: space between `if (` and `{` on new line)

Reply:
Don't know how I missed committing that. It should be all good now.